### PR TITLE
Misc cleanup of `seqr` chart.

### PR DESCRIPTION
**Pull request by bpblanken**
I noticed a few low-stakes things while going through the process of making the `hail-search` chart.

### CODE REVIEW

File: charts/seqr/README.md [hunk 1]
```diff
@@ -94,7 +94,6 @@ If you need to upgrade the redis version made available by this chart, you can d
 | fullnameOverride | string | `""` | Overrides the fully qualified name of the app, for use in templates |
 | image.pullPolicy | string | `"Always"` | The policy for pulling images |
 | image.repository | string | `"gcr.io/seqr-project/seqr"` | The docker image repository to pull images from |
-| imagePullSecrets | list | `[]` | If needed, you can provide secrets required to retrieve images |
```
Comment (bpblanken) [inline charts/seqr/README.md:97]: These are some `helm` provided charts that our deployment does not make use of.

File: charts/seqr/templates/_helpers.tpl [hunk 2]
```diff
@@ -56,7 +56,7 @@ Seqr environment shared between application and cron
 {{- define "seqr.environment" -}}
 envFrom:
   - configMapRef:
-      name: {{ .Release.Name }}-environment
```
Comment (bpblanken) [inline charts/seqr/templates/_helpers.tpl:59]: `seqr.fullname` and `.Release.Name` may be the same, but not guaranteed!  The `ConfigMap` name is denoted [here](https://github.com/broadinstitute/seqr-helm/blob/main/charts/seqr/templates/environment.yaml#L4)

File: charts/seqr/README.md (hunk 1 above)
Comment (sjahl) [reply charts/seqr/README.md:97]: when it comes to imagepullsecrets and tolerations, i'd actually argue that it's a bug that the chart isn't using them. Pull secrets and tolerations are both things that are common to need to set in various deployments, so the chart for public consumption should probably allow for it.

File: charts/seqr/templates/_helpers.tpl (hunk 2 above)
Comment (sjahl) [reply charts/seqr/templates/_helpers.tpl:59]: 👍

**Review by sjahl — COMMENTED** [review]
overall changes look fine to me -- i noted this inline, but i'd actually opt to update the chart to actually use the pull secrets and tolerations. Those are pretty common settings in various k8s environments

File: charts/seqr/README.md (hunk 1 above)
Comment (bpblanken) [reply charts/seqr/README.md:97]: yep that's also reasonable!

**Review by sjahl — APPROVED** [review]
thanks for adding in the missing values -- this looks good to me!

**Pull request merged**